sdk/logs-testing/src/test/java/io/opentelemetry/sdk/testing/assertj/LogAssertionsTest.java

```diff
@@ -109,7 +110,8 @@ void passing() {
                         attributeEntry("colors", "red", "blue"),
                         attributeEntry("conditions", false, true),
                         attributeEntry("scores", 0L, 1L),
-                        attributeEntry("coins", 0.01, 0.05, 0.1)));
+                        attributeEntry("coins", 0.01, 0.05, 0.1)))
+        .hasTotalAttributeCount(10);
```

Comment:
This is a tiny bit confusing because the number is pretty close to the actual number (8 keys, 13 values). What about  making it considerably larger for clarity -- like 999 or something?